### ISSUE TITLE
ISO duration formatting instead of seconds

### ISSUE THREAD

**Issue by arozans** [vit-project/vit#275]
**Describe the bug**
When defining duration uda in taskrc:
```
uda.myestimate.type=duration
uda.myestimate.label=Estimate
```
it's displayed correctly in the "next" report, eg: "PT3H".
In vit it's displayed without formatting, in seconds: "10800"

I want to have the same effect as in taskwarrior reports.

I've tried to write my own formatter, but sadly it doesn't seem to be picked up:
~/formatter/duration.py
```
from vit.formatter import Duration


class Duration(Duration):
    def format(self, obj, task):
        if not obj:
            return self.empty()
        formatted_duration = self.format_duration(obj)
        return (len(formatted_duration), self.markup_element(obj, formatted_duration))

    def format_duration(self, obj):
        return "FOO" # test value

    def markup_element(self, obj, formatted_duration):
        return (self.colorize(obj), formatted_duration)

```

Thanks.

**Comment by arozans** [comment]
Hmm, 
problem resolved itself somehow...
When I added first duration uda in taskwarrior few days ago via command "task xxx edit myestimate: 3h", it was saved like that:
```
2020-12-05 22:28:39 Myestimate set to '10800'.
``` 
But today, when i set this uda field in another task, it is saved like that:
```
2020-12-11 22:46:05 Myestimate set to 'PT2H'.                                  
```
First one is only displayed correctly in tw, not in vit. Second one is displayed properly in both apps.
Strange.

**Comment by thehunmonkgroup** [comment]
Unable to reproduce.

Used https://github.com/vit-project/vit/blob/2.x/scripts/generate-dummy-install.sh to generate test install

Added the following to ```.taskrc```:

```dosini
uda.myestimate.type=duration
uda.myestimate.label=Estimate

report.next.columns=id,start.age,project,description,urgency,myestimate
report.next.description=Most urgent tasks
report.next.filter=status:pending limit:page
report.next.labels=ID,Active,Project,Description,Urg,Est
report.next.sort=urgency-
```
Duration renders correctly in VIT:
![uda-duration](https://user-images.githubusercontent.com/43772/101959823-f547a080-3bba-11eb-908b-3bca841bc50d.png)

**Comment by arozans** [comment]
Thanks for your effort that you've put in reproducing this.
I think I found the reason for the inconsistency I saw.

Taskwarrior defines possible duration formats to use in commands here: https://taskwarrior.org/docs/design/task.html#type_duration

Accordignly to docs that formats:
`myestimate:1000s
`
is perfectly fine. 
So when I execute import command that way:
`task import tmp_task.txt`
and my file `tmp_task.txt` contains:
`{
  "id": 296,
  "myestimate": "1000s",
  "description": "bar",
  "entry": "20201219T235203Z",
  "modified": "20201220T182347Z",
  "project": "PROD_TOOLS",
  "start": "20201220T181907Z",
  "status": "pending",
  "uuid": "c0cf95b0-9a7a-42c7-9819-1824dc9aa32f",
  "urgency": 25
}
`
then taskwarrior renders it correctly:

![image](https://user-images.githubusercontent.com/16906186/102721745-1451db00-42fd-11eb-96c0-d904ecd39a9c.png)

vit on the other hand show it as raw values:
![image](https://user-images.githubusercontent.com/16906186/102722060-df468800-42fe-11eb-85b0-622f003e44ad.png)

Now, when I edit task with "mod" command:
`❯ task mod 296 myestimate:1000s`
it's rednered correctly in both taskwarrior reports and in vit (sane as in the first picture). So there must be some differences between `task modify` and `task import` inner workings?
